packages/salesforcedx-utils-vscode/test/unit/cli/orgOpenContainerResultParser.test.ts

```diff
@@ -15,16 +15,15 @@ import {
 
 // tslint:disable:no-unused-expression
 describe('force:org:open container parser', () => {
+  const orgOpenSuccessResult: OrgOpenSuccessResult = {
+    status: 0,
+    result: {
+      orgId: '00Dxx0000000123',
+      url: 'www.example.com',
+      username: 'test@example.com'
+    }
+  };
   it('should parse success info successfully', () => {
```

Comment:
[Nitpick] Put a newline between the `it(..)` methods to make them consistent. The spacing between the `it` clauses are not consistent. Prettier cannot help fix these kinds of formatting issues.